README.md

```diff
@@ -204,3 +206,8 @@ or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any addi
 ***
 # License
 [MIT License](LICENSE)
+
+# Data/Telemetry
```

Comment:
We must call out that it is disabled by default.

Reply:
My idea was to have the DOCS clear that telemetry will be on by default.  then make the code obey that as and when we are ready .   What I want to avoid is have the docs say that they are off by default, and then when we later turn the code on it's not super clear that you need to go an read the docs again (aka breaking change).   in this manner, the docs and "design" are all such that it is clearly messaged telemetry is ON by default.    it's a "bug" that we don't have the code behaving per spec yet.  In that fashion we can "fix the code" later and all be good.   It isnt a breaking change .   you like ?

---
In reply to: [335297148](https://github.com/microsoft/onnxruntime/pull/2109#discussion_r335297148) [](ancestors = 335297148)

Reply:
I thought we decided to leave the session option for this to be disabled by default in 1.0 release. The session option config should match the docs, right? Or I'm missing something?

Reply:
The docs should reflect the current state. To start, it will be OFF by default. When it's out of preview and ready to fully turn on, we will update the documentation to reflect that state. 

For the specific documentation, I suggest:
- main readme page mentions that telemetry collection is implemented in this project and to refer to the detailed data collection page. We don't need to change this documentation when default state changes. 
- data collection page includes privacy policy as well as the documentation on how to enable/disable
- C API page just links to data collection page. We can still have a section heading, but it will contain nothing but the link to the data collection page.

Reply:
change the docs to denote it is OFF by default during BETA, and to expect it to ON by default once it graduates from BETA (for transparency)

---
In reply to: [335777476](https://github.com/microsoft/onnxruntime/pull/2109#discussion_r335777476) [](ancestors = 335777476)

Reply:
I made this change  like you suggested, just a heads up that now we have C API specific docs embedded in the privacy statement.    If we ever have projections on these (like C# or WinRT) we might have to figure out where to put those docs.  I had the turn/on and off in the C_API docs since it was specific to the C_API.

Let me know what you think with the new doc layout !

---
In reply to: [335829030](https://github.com/microsoft/onnxruntime/pull/2109#discussion_r335829030) [](ancestors = 335829030)